code/modules/mob/living/living_emote.dm

```diff
@@ -112,6 +112,8 @@
 
 /datum/emote/living/deathgasp/play_sound_effect(mob/user, intentional, sound_path, sound_volume)
 	var/mob/living/carbon/human/H = user
+	if(!user)
+		return ..()
 	// special handling here: we don't want monkeys' gasps to sound through walls so you can actually walk past xenobio
 	playsound(user.loc, sound_path, sound_volume, TRUE, frequency = H.get_age_pitch(), ignore_walls = !isnull(user.mind))
```

Comment:
It is better to just replace the `H.get_age_pitch()` call here with something else if it ain't human.
If you scroll up you can see that other things can also use this emote.

Reply:
If it ain't human it falls back to generic `/datum/emote/proc/play_sound_effect`; it's not completely silent.
The base proc does use the same kind of age-pitch shift, but only for humans.
https://github.com/ParadiseSS13/Paradise/blob/8b57e54e71afada36fa2bfe2ddac62384a863f0f/code/datums/emote.dm#L258-L264
The `playsound` call is duplicated here (in /emote/living/) simply to add the extra `ignore_walls` snowflake

And before the refactor robots seemingly didn't audiably deathgasp at all?
https://github.com/ParadiseSS13/Paradise/blob/9dc65f5cf022411bbd66f504cc80b31c90f3ecc4/code/modules/mob/living/silicon/robot/emote.dm#L87-L89

Reply:
Ah I see. That logic is good to go then, thanks for the detailed reply!